### ISSUE TITLE
Error Detected While Trying To Enable Plugins On Windows 10

### ISSUE THREAD

**Issue by EvilDroidMax** [axvr/Vivid.vim#25]
I'm having problems trying to enable plugins on windows 10 gvim.
I can install them fine but when i try to enable them I get this error,

Error detected while processing function <SNR>2-enable_validate[2]..vivid#enable
[4]..<SNR>3_enable_plugins:

E919: Directory not found in 'packpath': "pack/*/opt/(plugin name)

**Comment by axvr** [comment]
Vivid is not yet compatible with Windows, however this is being worked on. 

I do have a problem though; I can't test it myself. The optimisations which I am working on should solve this problem and will be released by Saturday 14th October (3 days time). If I could mention (@EvilDroidMax) you in this issue when it is fixed, it would be great if you could test it for me.

**Comment by EvilDroidMax** [comment]
I got it working by setting set packpath+=~/.vim , it seems gvim on windows looks for vimfiles to load plugins and things but when you start it first time it actualy makes a .vim path (given the fact that I come from linux as my main OS I got confused)

Also a few things I would like to suggest.
It would be nice to have a way to autofill plugins name when you enable them for example. If I type PluginEnable light (Tab) it should autofill to lightline.

I'm fine with testing things since I decided to use this plugin as Plugin manager on windows (I use dein on linux) Going to try and see if I can get it to working without any problem

**Comment by axvr** [comment]
@EvilDroidMax I have added better Windows support for Vivid and very basic tab completion for commands (which will be improved a lot soon, it currently has many problems).

If you could check the Windows compatibility of this new version that would be great, thanks.

**Comment by EvilDroidMax** [comment]
The Completions works flawless as far as I tried.
Yesterday I tried to run some benchmark on windows and vivid was using more time to load the plugins comparing it to dein (made it so dein loads the plugins same way as vivid without delaying them. Basically both running plugins at start) .
I'm getting vivid on Linux to try some more benchmark (since I can't get StartupTime Plugin to work on gvim anymore)

Update: Did some benchmark on linux and it seems they load basically the same.

Vivid : 31 Plugins : 259 ms
Dein: 35 Plugins : 288

I think speed seems fine on linux side, going to try and get the benchmark working on windows again.
